src/SFML/Graphics/Color.cpp

```diff
@@ -68,6 +68,24 @@ a(alpha)
 
 
 ////////////////////////////////////////////////////////////
+Color::Color(Uint32 color) :
+r((color & 0xff000000) >> 24),
+g((color & 0x00ff0000) >> 16),
+b((color & 0x0000ff00) >> 8 ),
+a((color & 0x000000ff) >> 0 )
+{
+
+}
+
+
+////////////////////////////////////////////////////////////
+Uint32 Color::toInteger()
+{
+    return r << 24 | g << 16 | b << 8 | a;    
```

Comment:
Not 100% sure right now, but does this even work without casting, considering the components are `Uint8`?